.eslintrc.json

```diff
@@ -0,0 +1,20 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true,
+    "node": true
```

Comment:
We don't use this in the NodeJs environment.
About env setup for eslint, please refer to this link: https://eslint.org/docs/user-guide/configuring/language-options#specifying-environments

Reply:
Okay I'll remove them

Reply:
Oh I think I understand what you mean, I'll change it.